compose.yml

```diff
@@ -118,7 +121,9 @@ services:
             - MATOMO_DATABASE_HOST=db
             - MATOMO_DATABASE_DBNAME=matomo
             - MATOMO_DATABASE_USERNAME=root
-            - MATOMO_DATABASE_PASSWORD=$MW_DB_INSTALLDB_PASS
+            - MATOMO_DATABASE_PASSWORD_FILE=/run/secrets/db_root_password
```

Comment:
There is also no `MATOMO_DATABASE_PASSWORD_FILE` env https://github.com/matomo-org/docker so it won't work just like that

Reply:
It will work because [docker-entrypoint.sh#L10](https://github.com/matomo-org/docker/blob/3e696d5aa7869c243774658282c07307e65723a6/docker-entrypoint.sh#L10) allows to define all the environment variables using files.

Reply:
Thanks, I did not spot that, resolved